frontend/src/component/project/Project/ProjectSettings/ProjectActions/ProjectActionsTable/ProjectActionsModal/ProjectActionsForm/ProjectActionsFormStep/ProjectActionsFormStepSource/ProjectActionsFilterItem.tsx

```diff
@@ -211,15 +219,23 @@ export const ProjectActionsFilterItem = ({
             <StyledFilter>
                 <StyledFilterHeader>
                     <StyledInputContainer>
-                        <StyledInput
-                            label='Parameter'
+                        <Autocomplete
+                            freeSolo
+                            options={suggestions}
```

Comment:
Should we add a notice that this is autocompleted from the last event? I can imagine people being a bit confused about this autocomplete if they don't find what they're looking for or if after a refresh they see different options. If we have that possibility available in the component, then great, otherwise we can postpone this until later

Reply:
This is a good point and actually something that me and @nicolaesocaciu discussed. We decided to move forward with this for now, but perhaps there's an opportunity for improving this in a future PR.